test/blackbox-tests/test-cases/github7034.t/run.t

```diff
@@ -106,23 +106,16 @@ But when lang dune is 3.3 or higher the warning becomes an error:
     -short-paths
     -keep-locs))
   Leaving directory 'outer'
+
   $ dune build --root=outer
   Entering directory 'outer'
-  File "vendored/inner/inner.ml", line 6, characters 11-18:
```

Comment:
@gridbugs could you take a look at this test? Does this PR fix the issue?

Reply:
yep!